automatic/freecad/freecad.nuspec

```diff
@@ -51,6 +51,7 @@ Example: `choco install freecad --params "/InstallDir:'C:\FreeCAD' /NoShortcut"`
     <tags>freecad 3d cad modeling engineering admin</tags>
     <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/freecad</packageSourceUrl>
     <dependencies>
+      <dependency id="chocolatey" version="1.0.0" />
```

Comment:
This is the wrong version. It is the `2.x` version that has the correct algorithm.

AppVeyor upgraded to `2.2.2` automatically during the freecad installation (may be a bug in CLI).

Please update it to be (I think) `2.2.2`.

Reply:
So, this is using 1.0.0 for the Freecad-non7z packages, and 2.2.2 for the 7z package that depends on that. I'm unsure if it would be better to remove the dependency entirely for the install packages, but 1.0 seemed a reasonable compromise there - I assumed we didn't want to have a latest-version dependency where it's not actually needed?

Reply:
We do not want to have a dependency on Chocolatey CLI if we do not need to have one.

So in the end for this package, only the version that needs the dependency should add it, while the other versions should have no dependency on the CLI.